Resources/Prototypes/Entities/Mobs/Species/skeleton.yml

```
@@ -102,6 +102,7 @@
           probability: 0.5
   - type: FireVisuals
     alternateState: Standing
+  - type: FootPrints # WD EDIT
```

yamllint

[warning] wrong indentation: expected 4 but found 2

(indentation)